packages/@o3r/core/schematics/ng-update/v10.0/configuration.ts

```diff
@@ -0,0 +1,24 @@
+import type { Rule, Tree } from '@angular-devkit/schematics';
+import {getFilesWithExtensionFromTree} from '@o3r/schematics';
+
+const matchingRegex = /(Configuration)<\s+['"]strict['"]\s+>/g;
```

Comment:
I wonder it you can not remove `legacy` as well as there are both deprecated:

```suggestion
const matchingRegex = /(Configuration)<\s*['"](?:strict|legacy)['"]\s*>/g;
```

(not sure to understand why `\s+` instead of `\s*` as well)